tpl/cast/init_test.go

```diff
@@ -36,3 +38,11 @@ func TestInit(t *testing.T) {
 	require.True(t, found)
 	require.IsType(t, &Namespace{}, ns.Context.(func() interface{})())
 }
+
+func _TestNamespaceToJSON(t *testing.T) {
```

Comment:
It is a poor man's way of disable it. It is useful during development -- it prints a massive JSON to stdout, so not very helpful ... I may add some assertions to it and enable it. I _may_ do that.